frontend/src/register-login/PasswordStrength.css

```diff
@@ -0,0 +1,40 @@
+.PasswordStrength {
```

Comment:
Rename classes to css conventions, for ex. .PasswordStrength to .password-strength